string-manipulation-1/src/reverse-word.js

```diff
@@ -0,0 +1,8 @@
+/* exported reverseWord */
+function reverseWord(word) {
+  var newWord = '';
+  for (var i = 0; i < word.length; i++) {
+    newWord += word[word.length - 1 - i];
+  }
```

Comment:
You can simplify this by making the loop go in reverse. Start at the last index and count down until you hit zero